evalml/pipelines/components/estimators/regressors/arima_regressor.py

```diff
@@ -185,8 +203,10 @@ def fit(self, X, y=None):
 
         sp = self._get_sp(X)
         self._component_obj.sp = sp
+        self.last_X_index = X.index[-1] if X is not None else y.index
```

Comment:
This looks kind of weird....either the value is a value of an index...or it's an index?

Reply:
oops - will fix!